airflow/providers/docker/operators/docker.py

```diff
@@ -35,6 +35,15 @@
     from airflow.utils.context import Context
 
 
+def stringify(line: Union[str, bytes]):
+    """Make sure string is returned even if bytes are passed. Docker stream can return bytes."""
+    decode_method = getattr(line, 'decode', None)
```

Comment:
This looks a bit weird but is the cleanest way to make MyPy happy: https://github.com/python/mypy/issues/1424

Reply:
Yeah. Not the "bestest" of solutions so to speak. But it passed the tests too :)

Reply:
I had to do some similar shenanigans in the EKS tests module to get around the fact that the test fixtures can't accept parameters,  It's wonky, but that's Python.  All the rope you need to hang yourself in style. :tm:  LOL